python/ray/projects/schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "type": "object",
+  "properties": {
+    "name": {
+      "description": "The name of the project",
+      "type": "string"
+    },
+    "description": {
+      "description": "A short description of the project",
+      "type": "string"
+    },
+    "repo": {
+      "description": "The URL of the repo this project is part of",
+      "type": "string"
+    },
+    "cluster": {
+      "description": "Path to a .yaml cluster configuration file (relative to the project root)",
+      "type": "string"
+    },
+    "environment": {
+      "description": "The environment that needs to be set up to run the project",
+      "type": "object",
+      "properties": {
+        "dockerfile": {
+          "description": "Path to a Dockerfile to set up an image the project can run in (relative to the project root)",
```

Comment:
This mean we will build the docker file and then start ray cluster inside it? In that case, do we also support `Docker Image` that we can directly pull from?

Reply:
I added a "dockerimage" field which can be used for this purpose